doc/source/whatsnew/v1.1.4.rst

```diff
@@ -20,6 +20,7 @@ Fixed regressions
 - Fixed regression in :class:`RollingGroupby` with ``sort=False`` not being respected (:issue:`36889`)
 - Fixed regression in :meth:`Series.astype` converting ``None`` to ``"nan"`` when casting to string (:issue:`36904`)
 - Fixed regression in :class:`RollingGroupby` causing a segmentation fault with Index of dtype object (:issue:`36727`)
+- Fixed regression in :attr:`MultiIndex.is_monotonic_increasing` returning wrong results with ``"nan"`` in at least one of the levels (:issue:`37220`)
```

Comment:
Probably ``np.nan`` or ``NaN`` rather than the string ``"nan"``

Reply:
Thanks, this is better,